Resources/Textures/Clothing/Back/Duffels/hydroponics.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
-  "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/2e24b7af2221928e4b844a29408e821b60a5fe29",
+  "license": "CC0-1.0",
```

Comment:
Since the original is ShareAlike, are you allowed to do this?

Reply:
how did i mess that up